src/lib/linux-api/src/epoll.rs

```diff
@@ -0,0 +1,50 @@
+use crate::{bindings, const_conversions};
+use num_enum::{IntoPrimitive, TryFromPrimitive};
+
+bitflags::bitflags! {
+    /// Epoll create flags, as used with `epoll_create1`.
+    #[derive(Copy, Clone, Debug, Default, Eq, PartialEq)]
+    pub struct EpollCreateFlags: i32 {
+        const CLOEXEC = const_conversions::i32_from_u32(bindings::LINUX_EPOLL_CLOEXEC);
```

Comment:
We've generally been keeping the "type prefixes" for easier cross-referencing with man-pages etc; e.g. `EPOLL_CLOEXEC` here. I'm open to revisiting, but probably makes sense to be consistent one way or the other within the `linux-api` crate.

Reply:
No problem. My fault for trying to be fancy :)